### PR TITLE
Add tab check

### PR DESCRIPTION
**Pull request by josh-hadley**
- add a simple Python script that checks all .java files in the project for tab characters. The script prints the full path of each file found containing tabs and fails if any are found. The script can be run independently of CI.
- minor modifications to the Python workflow to link up the tab check script

### CODE REVIEW

File: py/tabcheck.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                    help="Force exit with status code 0 even if there are errors.")
+
+    args = ap.parse_args()
+    root = Path(args.ROOT_DIR)
+
+    if not root.exists():
+        print(f'{root.absolute()}: directory not found; aborting.')
+        return False
+
+    error_count = 0
```
Comment (markusicu) [inline py/tabcheck.py:32]: Why do you count the errors when you only return a boolean?

**Review by markusicu — APPROVED** [review]
One minor comment.

Otherwise lgtm -- please **squash-and-merge** when ready.

File: py/tabcheck.py (hunk 1 above)
Comment (josh-hadley) [reply py/tabcheck.py:32]: Ah, good point. I was originally going to return the error_count (for a non-zero exit code) but changed my mind later on without modifying that bit. I can change that to "errors_found" and treat it as a boolean if you prefer. I don't think it makes much difference.

Comment (markusicu) [reply py/tabcheck.py:32]: I would like that. I can quickly re-approve.

Comment (josh-hadley) [reply py/tabcheck.py]: done in https://github.com/unicode-org/unicodetools/pull/207/commits/af7a0e13c7d04b1a16157936971ab2348b1e047a

**Review by markusicu — APPROVED** [review]
lgtm
please **squash-and-merge**

**Pull request merged**